e2e/tests/pages/home.test.ts

```diff
@@ -23,53 +23,4 @@ test.describe('Home', () => {
   test('has the correct level 2 heading', async ({ page }) => {
     await expect(page.getByRole('heading', { level: 2, name: 'Updates' })).toBeVisible()
   })
-
-  test.describe('navigation', () => {
-    test.describe('@tablet-and-desktop', () => {
-      test('navigation is visible', async ({ page }) => {
-        await expect(page.getByRole('navigation', { name: 'Menu' })).toBeVisible()
-      })
-    })
-
-    test.describe('@mobile', () => {
-      test('navigation is hidden', async ({ page }) => {
-        await expect(page.getByRole('navigation', { name: 'Menu' })).toBeHidden()
-      })
-
-      test('clicking menu toggles navigation menu', async ({ page }) => {
-        await page.getByRole('button', { name: 'Menu' }).click()
-
-        await expect(page.getByRole('navigation', { name: 'Menu' })).toBeVisible()
-        await expect(page.getByRole('button', { name: 'Menu' })).toHaveAttribute('aria-expanded', 'true')
-
-        await page.getByRole('button', { name: 'Menu' }).click()
-
-        await expect(page.getByRole('navigation', { name: 'Menu' })).toBeHidden()
-        await expect(page.getByRole('button', { name: 'Menu' })).toHaveAttribute('aria-expanded', 'false')
-      })
-
-      test.describe('@visual-regression', () => {
-        test('matches the saved screenshot', async ({ page }) => {
-          await page.getByRole('button', { name: 'Menu' }).click()
-          await expect(page.getByRole('document')).toHaveScreenshot(
-            'mobile-menu-expanded-screenshot.png',
-          )
-        })
-      })
-    })
-  })
-
-  test.describe('@visual-regression', () => {
-    test('the start button matches the saved screenshot on hover', async ({ page }) => {
-      const startButton = page.getByRole('button', { name: 'Get started' })
-      await startButton.hover()
-      await expect(startButton).toHaveScreenshot('start-button-hover.png')
-    })
-
-    test('the start button matches the saved screenshot on focus', async ({ page }) => {
-      const startButton = page.getByRole('button', { name: 'Get started' })
-      await startButton.focus()
-      await expect(startButton).toHaveScreenshot('start-button-focus.png')
-    })
-  })
```

Comment:
As navigation is a global component i wanted to make its own test and use the page-list.json to loop over and test the navigation states